components/Nav.js

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+export default function Nav() {
+  return (
+    <NavContainer>
+      <nav>
+        <Link href="/lyrics">
+          <A>Lyrics</A>
+        </Link>
```

Comment:
Here you could add another page, which would be the homepage (with href="/") so you can also return to the landing page when you are on the lyrics page :)